flow/Makefile

```diff
@@ -241,26 +241,24 @@ export RTLMP_RPT_FILE ?= partition.txt
 export RTLMP_BLOCKAGE_FILE ?= $(OBJECTS_DIR)/rtlmp/partition.txt.blockage
 
 #-------------------------------------------------------------------------------
-ifeq (, $(strip $(NPROC)))
+ifeq (,$(strip $(NUM_CORES)))
   # Linux (utility program)
-  NPROC := $(shell nproc 2>/dev/null)
+  NUM_CORES := $(shell NUM_CORES 2>/dev/null)
```

Comment:
`nproc` is a bin util. It can't be replaced here.

Reply:
Oops. Well spotted. Fixed.